lib/db/memory.js

```diff
@@ -84,10 +84,12 @@ MemoryDB.prototype.getOps = function(collection, id, from, to, options, callback
   if (typeof callback !== 'function') throw new Error('Callback required');
   util.nextTick(function() {
     var opLog = db._getOpLogSync(collection, id);
-    if (to == null) {
-      to = opLog.length;
+    if (!from) from = 0;
```

Comment:
ℹ️ This defaulting matches the behavior of sharedb-mongo
https://github.com/share/sharedb-mongo/blob/0013d15df717ac8af32aa7847ed64bca96be7f66/index.js#L632-L633